examples/material_gallery/lib/demo/list_demo.dart

```diff
@@ -0,0 +1,195 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'package:flutter/rendering.dart';  // TBD remove this
+
+enum ListDemoItemSize {
+  oneLine,
+  twoLine,
+  threeLine
+}
+
+class ListDemo extends StatefulComponent {
+  ListDemoState createState() => new ListDemoState();
+}
+
+class ListDemoState extends State<ListDemo> {
+  final GlobalKey<ScaffoldState> scaffoldKey = new GlobalKey<ScaffoldState>();
+  List<String> items = [];
+  ListDemoItemSize itemSize = ListDemoItemSize.threeLine;
+  bool isDense = true;
+  bool showAvatar = true;
+  bool showIcon = false;
+  bool reverseSort = false;
+
+  void initState() {
+    super.initState();
+    items = <String>[
```

Comment:
I assume that there's some tradeoff about work done when the class is loaded vs when it's instantiated the first time. In this case I was not considering any tradeoffs :-). I've moved the value to the initializer.